.github/release-drafter-main.yml

```diff
@@ -9,16 +9,20 @@ categories:
       - 'bugfix'
       - 'bug'
   - title: '🧰 Maintenance'
-    label: 
+    label:
       - 'chore'
       - 'dependencies'
   - title: '📖 Documentation'
-    label: 
+    label:
       - 'documentation'
 
 exclude-labels:
   - 'skip-release-drafter'
 change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+name-template: 'Play $NEXT_MINOR_VERSION'
+tag-template: '$NEXT_MINOR_VERSION'
+filter-by-commitish: true
+commitish: main
```

Comment:
could this be dynamic based of the branch release drafter is triggered from ?

Reply:
It cant. Inside these config files you can access normal `GitHub` scope. the dynamic stuff in here is templated by the action itself, not by GitHub.

Reply:
I don't think release drafter can do what we want.

Ultimately thought i don't think this is _too_ much duplication.

For every release branch we will have to duplicate this file, and change this one line.
then in the actual branch, change (and rename) `on-<branch>_release-drafter.yml` and change one line in there.

Compared to some of the other guard conditions that some workflows have such as `if: github.ref != 'refs/heads/main' && github.ref != 'refs/heads/5.x.x'` especially.